aiida/backends/tests/dataclasses.py

```diff
@@ -127,12 +128,12 @@ def test_reload_singlefiledata(self):
 
         self.assertEquals(a.get_folder_list(), [basename])
 
-        with open(a.get_abs_path(basename)) as f:
+        with io.open(a.get_abs_path(basename), encoding='utf8') as f:
```

Comment:
possibly rename `f` to `fhandle` (or similar to make the linter happy) while being at it?